test/functional/apps/dashboard/group4/index.ts

```diff
@@ -25,9 +25,10 @@ export default function ({ getService, loadTestFile }: FtrProviderContext) {
     before(loadLogstash);
     after(unloadLogstash);
 
+    loadTestFile(require.resolve('./dashboard_empty'));
     loadTestFile(require.resolve('./dashboard_save'));
-    loadTestFile(require.resolve('./dashboard_time'));
-    loadTestFile(require.resolve('./dashboard_listing'));
-    loadTestFile(require.resolve('./dashboard_clone'));
+    // loadTestFile(require.resolve('./dashboard_time'));
+    // loadTestFile(require.resolve('./dashboard_listing'));
+    // loadTestFile(require.resolve('./dashboard_clone'));
```

Comment:
Is there a reason these other tests are commented out?

Reply:
🤭

Reply:
I commented them out just for testing purposes. That would have been **bad**. Thank you for catching this.